### DISCUSSION TITLE
`magit-blame` default face doesn't use my default font

### DISCUSSION THREAD

**Discussion by aadcg** [magit/magit#4696] [Support] [answered]
The title speaks for itself, and so does the screenshot below. 

Any ideas?

![2022-05-25_15:35:45](https://user-images.githubusercontent.com/45483512/170288778-f66787a2-4806-42be-a59e-c4d7bba1c88c.png)

**Comment by tarsius** [comment, marked as answer]
> The title speaks for itself, and so does the screenshot below.

No it doesn't. There is no "magit-blame default face" and your screenshot looks perfectly okay.

> Any ideas?

You don't like the look, so customize the faces in the `magit-blame` group.

**Comment by aadcg** [comment]
For those who use the `modus-themes` and wonder about this, please see the [reply I got from the developer of the themes](https://lists.sr.ht/~protesilaos/modus-themes/%3C874k1d9utu.fsf%40gmail.com%3E).
